README.md

```diff
@@ -1 +1,50 @@
-# ramen
+# Ramen
+
+Ramen is a **disaster-recovery orchestrator** for stateful applications across
+a set of peer kubernetes clusters that,
+
+- Preserve state using persistent volumes provisioned using [ceph-csi](https://github.com/ceph/ceph-csi/)
+- Uses the CSI [storage replication extensions](https://github.com/csi-addons/volume-replication-operator)
+  , which enable ceph-csi to replicate state across peer clusters
+- Are deployed and managed using [open-cluster-management](https://open-cluster-management.io/concepts/architecture/)
```

Comment:
"Is deployed ..." instead of "Are deployed"?  Given that this is not a Ramen functionality, a better place for this sentence may be in the opening paragraph, with something like: "that is typically deployed ...".  Also, may be good to define the OCM acronym here, as it is being used below.